cpp/modmesh/view/RManager.cpp

```diff
@@ -117,57 +117,46 @@ void RManager::setUpMenu()
     // NOTE: All menus need to be populated or Windows may crash with
     // "exited with code -1073740791".  The reason is not yet clarified.
 
+    m_fileMenu = m_mainWindow->menuBar()->addMenu(QString("File"));
     {
-        m_fileMenu = m_mainWindow->menuBar()->addMenu(QString("File"));
-
-        {
-            auto * action = new RAction(
+        m_fileMenu->addAction(
```

Comment:
Simplify the code to create of menu item.

Reply:
Sweet !
The code looks more elegance!